extensions/amp-connatix-player/amp-connatix-player.md

```diff
@@ -1,19 +1,5 @@
-<!--
-  1. Change "category" below to one of:
-       ads-analytics
-       dynamic-content
-       layout
-       media
-       presentation
-       social
-
-  2. Remove any of the "formats" that don't apply.
-     You can also add the "ads" and "stories" formats if they apply.
-
-  3. And remove this comment! (no empty lines before "---")
--->
 ---
-$category: media
+$category@: media
```

Comment:
@cvializ Is the `@` sign necessary? If so, I'll fix the generator.

Reply:
I think it might be, but I don't know enough about the format to say for certain. Do you know if the format is documented somewhere?